assets/prototype/domain/eventEditor/data/queries/datetimes/useDatetimeIds.ts

```diff
@@ -0,0 +1,10 @@
+import useDatetimes from './useDatetimes';
+import { Datetime, EntityId } from '../../types';
+
+const useDatetimeIds = (): EntityId[] => {
+	const datetimes: Datetime[] = useDatetimes();
+
+	return datetimes.map(({ id }: Datetime) => id);
```

Comment:
We don't need `: Datetime` part here, it is aware of the underlying data:

![image](https://user-images.githubusercontent.com/1477453/71372257-81347700-25bc-11ea-9137-1f98d85a739f.png)

Reply:
true